x-pack/plugins/secops/server/graphql/sources/schema.test.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { graphql } from 'graphql';
+import { addMockFunctionsToSchema, makeExecutableSchema } from 'graphql-tools';
+
+import { rootSchema } from '../../../common/graphql/root/schema.gql';
+import { Logger } from '../../utils/logger';
+import { sourcesSchema } from './schema.gql';
+import { getSourceQueryMock, mockSourceData } from './source.mock';
+
+const testCaseSource = {
+  id: 'Test case to query basic information from source',
+  query: `
+		query SourceQuery($sourceId: ID!) {
+			source(id: $sourceId) {
+				id
+				configuration {
+					fields {
+						host
+					}
+				}
+			}
+		}
+	`,
+  variables: { sourceId: 'default' },
+  context: {
+    req: {
+      payload: {
+        operationName: 'test',
+      },
+    },
+  },
+  expected: {
+    data: {
+      source: {
+        ...mockSourceData,
+      },
+    },
+  },
+};
+
+// const testCaseAllSources = {
```

Comment:
i would prefer to not check in commented-out code

Reply:
I will remove it